core/app/assets/styles/tokens/_spacing.scss

```diff
@@ -0,0 +1 @@
+$eps-spacer: 0.5rem;
```

Comment:
What is it used for?

Reply:
To make a good mathematical base spacing unit for multiplication